internal/integratedservices/services/securityscan/operator.go

```diff
@@ -352,3 +358,32 @@ func (op *IntegratedServiceOperator) applyLabelsForSecurityScan(ctx context.Cont
 
 	return nil
 }
+
+func (op IntegratedServiceOperator) createDefaultPolicyBundles(ctx context.Context, anchoreClient anchore.AnchoreClient) error {
+	files, err := ioutil.ReadDir(op.config.Anchore.PolicyPath)
+	if err != nil {
+		return errors.WrapIfWithDetails(err, "failed to read policy bundle directory", op.config.Anchore.PolicyPath)
```

Comment:
**Recommended**: error details should be key-value pairs.

Same at
- https://github.com/banzaicloud/pipeline/pull/3166/files?file-filters%5B%5D=.alpine&file-filters%5B%5D=.debug&file-filters%5B%5D=.dist&file-filters%5B%5D=.go&file-filters%5B%5D=.json&file-filters%5B%5D=.local&file-filters%5B%5D=.md#diff-a740f77bfb024386ae1072b3d2f46cf2R374
- https://github.com/banzaicloud/pipeline/pull/3166/files?file-filters%5B%5D=.alpine&file-filters%5B%5D=.debug&file-filters%5B%5D=.dist&file-filters%5B%5D=.go&file-filters%5B%5D=.json&file-filters%5B%5D=.local&file-filters%5B%5D=.md#diff-a740f77bfb024386ae1072b3d2f46cf2R379
- https://github.com/banzaicloud/pipeline/pull/3166/files?file-filters%5B%5D=.alpine&file-filters%5B%5D=.debug&file-filters%5B%5D=.dist&file-filters%5B%5D=.go&file-filters%5B%5D=.json&file-filters%5B%5D=.local&file-filters%5B%5D=.md#diff-a740f77bfb024386ae1072b3d2f46cf2R384

Reply:
I'm on it. Nice catch.